scripts/predicting_the_past_import.py

```diff
@@ -442,8 +478,8 @@ def select_region(in_list, region):
     for row in in_list:
         country = row[0]
         if country.upper() in regions[region.upper()]:
-            capacity = row[3]
-            start_date = row[10]
+            capacity = row[10]
+            start_date = row[8]
```

Comment:
What are these numbers? Why are these numbers chosen? 
Magic numbers are popping up all over the place which makes it hard to figure out what your code is actually doing (maybe you didn't introduce this issue but it should be fixed, I think). 

In fact, it looks like this behavior already existed and you've simply updated the numbers to work with other changes you've made. *Fine*, as long as tests are passing. But consider this: You could save time in the future by making this assignment predictable and automatic AND improve the readability of your code. As well as making it easier to maintain.